src/runapi.js

```diff
@@ -0,0 +1,61 @@
+const apiCommands = require("./tdx-api-commands");
+
+function objectToListMapper(obj) {
+  return Object.keys(obj).sort().map((element) => ([element, Number(element)]));
+}
+
+function stringifyArgumentValue(args, keyList) {
```

Comment:
Does this return something new or modify its input?

Reply:
Modifies the input.